packages/serverless-component/__mocks__/@sls-next/cloudfront.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
Could we avoid mocking `@sls-next/cloudfront`. I think this is implementation detail that we shouldn't be mocking even if is a different package.

Reply:
We cover AWS API calls with tests for the package ([`cache-invalidation.test.ts`](https://github.com/danielcondemarin/serverless-next.js/blob/ac809ee970981c9b9e3229cb29cd0885566b74e9/packages/cloudfront/tests/cache-invalidation.test.ts)). Do you think we should also mock the `aws-sdk` directly inside `serverless-component`? If so, should I just add a CloudFront mock inside `serverless-plugin/__mocks__/aws-sdk.js` or use a custom mock?

Reply:
Ah I see aws-sdk isn't mocked in the serveless-component. Nevermind, this is good for now, I'll look at tidying the tests up a bit later 👍